session_0/fizz_buzz.rb

```diff
@@ -37,5 +37,20 @@
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
 def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
+  if(n<0||x<=0||y<=0)
+    raise ArgumentError.new
+  end 
+  ar = Array.new(n)
+  for i in 0...ar.size do
```

Comment:
Do you think map function and ranges can be of use here? 

https://www.tutorialspoint.com/ruby/ruby_ranges.htm
https://www.rubyguides.com/2018/10/ruby-map-method/